docs/src/main/docs/tracing/01_Tracing.adoc

```diff
@@ -16,42 +16,44 @@
 
 ///////////////////////////////////////////////////////////////////////////////
 
-= Tracing Support
-:description: Helidon Reactive WebServer Tracing Support
-:keywords: helidon, reactive, reactive streams, reactive java, reactive webserver
+= Tracing
+:description: Helidon Tracing Support
+= :keywords: helidon, tracing
 
 == Tracing Support
-Helidon includes Zipkin support for OpenTracing. When enabled, the WebServer
- sends its tracing events to Zipkin.
+Helidon includes support for tracing through `OpenTracing` APIs. Tracing
```

Comment:
... "the" `OpenTracing` APIs.

Reply:
Consider adding a link to https://opentracing.io/, if that's the reference here.

Reply:
Added the link